firehose-fuel/devel/fuelfire/fuelfire.yaml

```diff
@@ -6,8 +6,11 @@ start:
     - relayer
   flags:
     # Specifies the path to the fuel-node binary.
-    reader-node-path: "./../../../../fullstack/fuel-core-with-firehose-node/fuel-core/target/debug/fuel-core"
+    reader-node-path: "./../../../firehose-extract/target/debug/firehose-extract"
```

Comment:
We should probably use release build here

Reply:
We can include this in the issue where we will add ci.yml ?

Reply:
Here it is: https://github.com/FuelLabs/fuel-subgraph/issues/12